mgr/src/cmds/pools.cr

```diff
@@ -1,79 +1,345 @@
 require "./helpers"
-require "./cli_table"
+require "./pool_create_parser"
+require "./gluster_volume_parser"
 
-command "pool.create", "Create the Kadalu Storage Pool" do |parser, _|
-  parser.banner = "Usage: kadalu pool create NAME [arguments]"
+struct PoolArgs
+  property status = false, detail = false, name = "", pool_id = "", no_start = false,
+    distribute = false, auto_add_nodes = false,
+    node_maps = Hash(String, String).new, volfiles_separator = " "
+end
+
+class Args
+  property pool_args = PoolArgs.new
+end
+
+command "pool.create", "Kadalu Storage Pool Create" do |parser, args|
+  parser.banner = "Usage: kadalu pool create NAME TYPE STORAGE_UNITS [arguments]"
+  parser.on("--no-start", "Don't start the pool upon creation") do
+    args.pool_args.no_start = true
+  end
+  parser.on("--pool-id=ID", "Set pool-id to import a pool") do |pool_id|
+    args.pool_args.pool_id = pool_id
+  end
+  parser.on("--distribute", "Create distributed Pool") do
```

Comment:
Documentation on usage of '--distribute`

Reply:
Not yet implemented in API side. I will remove this from here and add to separate PR. The idea of this flag is to make the user aware that the pool that is going to be created is distributed. Current behaviour is to automatically decide if more storage units are provided than the Replica/Disperse count.